modules/virt-adding-static-public-key-cli.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-ssh.adoc
+
+:_content-type: PROCEDURE
+[id="virt-adding-static-public-key-cli_{context}"]
+= Adding a static public key from the command line
+
+Add a static public key to a virtual machine (VM) by adding the secret and its propagation method to the `VirtualMachine` manifest.
+
+The SSH key is assigned to the VM as generated cloud-init metadata. This method separates the access credentials from the application data in the cloud-init user data. This method does not affect cloud-init user data.
+
+.Prerequisites
+
+* You created a secret containing the SSH public key in the same project as the VM.
+* The VM has never booted.
+
+.Procedure
+
+. Add the secret and the `configDrive` propagation method to the `VirtualMachine` manifest as in the following example:
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
```

Comment:
I would be surprised if this VM definition would be valid.
I vote for using a minimal RHEL VM here, agreed with QE.
I would start with https://drive.google.com/file/d/19Sf3XGWPnvi4qoJAQp2oq4oUdYN8JCv7/ and maybe it can be even shorter in 4.14.

But this comment is not related to the intention of this PR, so it should not block this PR.

Reply:
Is it OK to include the 'secret' part of the YAML too? It's a much more elegant solution to create the VM and secret at the same time, instead of creating a separate secret CR.

Reply:
Sorry, while writing this comments I was not aware that this PR is targeting 4.13, while the example requires 4.14. Can we have a separate jira story to adjust this example in 4.15 ?
- This would give us the time to ensure that QE is covering this example, too.
- In addition, there might be other places in the documentation
- Instancetypes would be out of TP and fully supported

Reply:
https://issues.redhat.com/browse/CNV-30930 will track this idea.

Reply:
On second thought, it would be simpler to target this for 4.14. I will update the Jira.

Reply:
unfortunately the new VM workflow used by CNV-21234  will be only TP in 4.14, and GA only in 4.15

Reply:
OK. What can I use for 4.14?

Reply:
Or would it be acceptable for me to replace the `dataVolumeTemplates`, `instancetype`, and `preference` stanzas with "..."?

Reply:
I think that it might be simpler just to use YAML examples instead of working YAMLs. I separated the Secret YAML and VM YAML.